test-network/prometheus-grafana/README.md

```diff
@@ -0,0 +1,47 @@
+# Description
+
+This sample provides an environment to display and capture metrics from the test-network in real time. It consists of a docker-compose file that starts a prometheus and grafana server setup configured to collect and display metrics for the test network.
+
+# Requirements
+
+This sample has been tested and is meant to be used on **linux** in order to fully benefit from its capabilities.
```

Comment:
I was able to successfully test on my MacOS-intel machine.

Reply:
where you able to bring up the cadvisor or did you have to change the image, did you then see all container metrics( cause I think by changing the image the labels change and you need to update the queries to see docker containers metrics)... all the other fabric metrics are fine so I added that works on mac but needs few tweak to show docker container metrics

Reply:
Your right... I was not able to see docker metrics. Only Fabric. Most people will probably run this on Mac, so that is worth saying.